spec/rails_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'capybara/rspec'
 require 'capybara/rails'
 require 'support/attributes'
+require 'support/test_helpers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.